session-helper/flatpak-session-helper.c

```diff
@@ -423,11 +423,14 @@ name_owner_changed (GDBusConnection *connection,
     {
       GHashTableIter iter;
       PidData *pid_data = NULL;
+      gpointer value = NULL;
       GList *list = NULL, *l;
 
       g_hash_table_iter_init (&iter, client_pid_data_hash);
-      while (g_hash_table_iter_next (&iter, NULL, (gpointer *)pid_data))
+      while (g_hash_table_iter_next (&iter, NULL, &value))
         {
+          pid_data = value;
+
```

Comment:
Looks good, but now I would move the pid_data declaration into the loop body.

Reply:
> but now I would move the pid_data declaration into the loop body

I did that at first myself, but the same variable is used in a vaguely similar way in another loop a little further down, so I would have had to declare two variables of the same name and type inside two different loops. I decided I preferred it this way.